masterlist.yaml

```diff
@@ -5062,33 +5055,12 @@ plugins:
       - <<: *patchProvided
         subs: [ 'Campfire - Complete Camping System' ]
         condition: 'active("Campfire.esm") and not active("Archery_Campfire_Patch.esp")'
-    clean:
-      # version: 2.5
-      - crc: 0xB2AD660F
-        util: 'SSEEdit v4.0.2'
-      # version: 2.4
-      - crc: 0x0DDC51B6
-        util: 'SSEEdit v4.0.2'
```

Comment:
```yaml
  - name: 'ArcheryV2.esp'
    dirty:
      # version: 2.4
      - <<: *quickClean
        crc: 0x0DDC51B6
        util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 4
      # version: 2.5
      - <<: *quickClean
        crc: 0xB2AD660F
        util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 4
```

ESL Version is incorrectly flagged as an `ESL`.

```
[00:00] Saving: ArcheryV2.esp.save.2019_06_15_02_38_13
[00:00] Error saving ArcheryV2.esp.save.2019_06_15_02_38_13: Record 1stPersonWarBow [STAT:FE00037C] has invalid ObjectID 02F37C for a light module. You will not be able to save this file with ESL flag active
[00:00] Errors have occured. At least one file was not saved.
```

Reply:
That invalid `ESL` flag will be picked up by LOOT.

![image](https://user-images.githubusercontent.com/1944639/59546184-20e3bf00-8f21-11e9-9d34-9cdaf36acd5d.png)